op-guide/binary-deployment.md

```diff
@@ -1,83 +1,197 @@
 ---
-title: TiDB Binary Deployment
+title: Deploy TiDB Using the Binary
 category: deployment
 ---
 
-# Binary Deployment
+# Deploy TiDB Using the Binary
 
 ## Overview
 
-A complete TiDB project contains PD, TiKV, TiDB. The start-up sequence is PD -> TiKV -> TiDB.
+A complete TiDB cluster contains PD, TiKV and TiDB. To start the database service, follow the order of PD -> TiKV -> TiDB. To close the database service, follow the reverse order of start-up.
```

Comment:
To close the database service, follow the reverse order of start-up.
->
To stop the database service, follow the order of stopping TiDB -> TiKV -> PD.